apps/arena/lib/arena/game_socket_handler.ex

```diff
@@ -37,7 +37,12 @@ defmodule Arena.GameSocketHandler do
         GameUpdater.attack(state.game_pid, state.player_id, skill)
 
       %{action_type: {:move, %{direction: direction}}, timestamp: timestamp} ->
-        GameUpdater.move(state.game_pid, state.player_id, {direction.x, direction.y}, timestamp)
+        GameUpdater.move(
```

Comment:
Beware with your formatting, you didn't make any change here to actually make a formatting. Also, the formatting might not be using the same rules

Reply:
There was a change but I reverted it. However the command `mix format` didn't change it again.